src/main/java/uk/gov/hmcts/ccd/domain/model/definition/RoleToAccessProfileDefinition.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.ccd.domain.model.definition;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class RoleToAccessProfileDefinition {
+
+    private String caseTypeId;
+    private Boolean disabled;
+    private Boolean readOnly;
```

Comment:
```suggestion
    private boolean disabled;
    private boolean readOnly;
```

Reply:
we need this to be Boolean Objects, we have different functionality if value doesn't exists